### ISSUE TITLE
Command to show all blacklists and active suspensions in the server 

### ISSUE THREAD

**Issue by Deaththth** [ewang2002/ToogaBooga#112]
Only show the info needed:
For blacklist only show ign and blacklistid
For suspension show ign/userid and suspension end

**Comment by saifalushi1** [comment]
who do we want able to use this command?

**Comment by ewang2002** [comment]
Probably helpers, securities, moderators, raid leaders (not almost leaders) and above

**Comment by saifalushi1** [comment]
ok im just tryna figure out what variables you are using like "realmname" to run queries to the db.

**Comment by ewang2002** [comment]
See if the other files help — a lot of our code essentially is just copied from other code files. My time the next few days is severely limited due to school; see if @dmansdman can help

**Comment by saifalushi1** [comment]
wait you already have a var for this 
* A list of all blacklisted users.
         *
         * @type {IBlacklistedUser[]}
         */
        blacklistedUsers: IBlacklistedUser[];

**Comment by saifalushi1** [comment]
im gonna write up a dummy command to see what it returns and if anything just tweak the returned data

**Comment by ewang2002** [comment]
Yeah we have that but you need to make it readable for the general user — using an embed or something.

**Comment by saifalushi1** [comment]
ok ill discuss with dman to see if it makes more sense to use that or just run a .findAll query

**Comment by ewang2002** [comment]
@saifalushi1 if you want to add a show all suspend command, go for it
